lib/http.js

```diff
@@ -125,7 +125,41 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   // update keystore config
   app.post(
     `${routes.keystore}`,
-    _authorizeZcapInvocation(),
+    _authorizeZcapInvocation({
+      getExpectedTarget: async ({req}) => {
```

Comment:
JFYI, I believe this is also an acceptable syntax:

```suggestion
      async getExpectedTarget({req}) {
```